oshc/main/static/main/css/app.css

```diff
@@ -154,13 +155,23 @@ a {
   padding: 0 50px;
 }
 
+iframe {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  border: 0;
+}
+
```

Comment:
we already have a new line over there. Having two consecutive new lines is not a good practice.